CMakeLists.txt

```diff
@@ -85,20 +90,22 @@ if(BUILD_TESTS OR UPDATE_SAMPLES)
     find_package(RapidJSON CONFIG REQUIRED)
 
     add_library(todaygraphql
-       ${CMAKE_BINARY_DIR}/TodaySchema.cpp
        Today.cpp
+       TodaySchema.cpp
```

Comment:
I believe it should be `${CMAKE_CURRENT_BINARY_DIR}/TodaySchema.cpp`, to support out-of-source builds.